### PR TITLE
[CWS] Allow syscall tester to be built with gcc

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Allow using gcc to build CWS embedded syscall tester.

### CODE REVIEW

File: tasks/security_agent.py [hunk 1]
```diff
@@ -253,14 +253,18 @@ def ninja_c_syscall_tester_common(nw, file_name, build_dir, flags=None, libs=Non
     if static:
         flags.append("-static")
 
+    additional_flags = [f"-isystem/usr/include/{uname_m}-linux-gnu"]
+    if compiler == "clang":
+        additional_flags.append(f"-D__{uname_m}__")
```
Comment (paulcacheux) [inline tasks/security_agent.py:258]: why is this required only for clang ?

Comment (paulcacheux) [reply tasks/security_agent.py:258]: yeah from what I tested you can actually remove completely this define in and it should work perfectly fine

Comment (lebauce) [reply tasks/security_agent.py:258]: That's good to know. Thanks for having tested it !

**Review by paulcacheux — APPROVED** [review]

**Review by spikat — APPROVED** [review]

**Review by gjulianm — APPROVED** [review]
